src/CachePlugin.php

```diff
@@ -36,11 +42,13 @@ class CachePlugin implements Plugin
 
     /**
      * @param CacheItemPoolInterface $pool
+     * @param StreamFactory          $streamFactory
      * @param array                  $options
      */
-    public function __construct(CacheItemPoolInterface $pool, array $options = [])
+    public function __construct(CacheItemPoolInterface $pool, StreamFactory $streamFactory, array $options = [])
```

Comment:
Wouldn't it kill the idea behind the options array and options resolver? Or did we reach the maximum number of possible options?